.changeset/modern-scissors-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-development-kitchen': major
```

Comment:
if this only adds a `size` option, i think it's a minor?

Reply:
I went with major because a) the `size` prop is required, and b) this PR also renames an existing field. Which I guess makes it not backwards compatible

Reply:
(we're not actually using this component yet)